packages/message-kit/src/lib/skills.ts

```
@@ -0,0 +1,265 @@
+  text: string,
+  skills: SkillGroup[],
+  context: HandlerContext,
+) {
+  // Use the custom text parameter
+  let conversation = context.conversation;
+  try {
+    let skillCommand = findSkill(text, skills);
+    const extractedValues = parseSkill(text, skills);
+    if ((text.startsWith("/") || text.startsWith("@")) && !extractedValues) {
+      console.warn("Command not valid", text);
+    } else if (skillCommand && skillCommand.handler) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)